From c434f2f0462b9839f8e3ee47c97006ff6f046247 Mon Sep 17 00:00:00 2001
From: Myron Stowe <myron.stowe@redhat.com>
Date: Thu, 10 Jul 2014 01:03:14 -0400
Subject: [acpi] hotplug/pci: Get rid of unused constants in acpiphp.h

Message-id: <20140710010314.22146.89409.stgit@gir.stowe>
Patchwork-id: 85587
O-Subject: [RHEL7.1 PATCH 048/105] ACPI / hotplug / PCI: Get rid of unused constants in acpiphp.h
Bugzilla: 1114228
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1114228
Upstream Status: c38f82cf1b7dc8aad2bce8e30113fd6aa3159dab

commit c38f82cf1b7dc8aad2bce8e30113fd6aa3159dab
Author: Mika Westerberg <mika.westerberg@linux.intel.com>
Date:   Sat Jul 13 23:27:26 2013 +0200

    ACPI / hotplug / PCI: Get rid of unused constants in acpiphp.h

    Drop some unused symbols from acpiphp.h and redefine SLOT_ENABLED
    (which is the only slot flag now) as 1.

    [rjw: Redefinition of SLOT_ENABLED, changelog]
    Signed-off-by: Mika Westerberg <mika.westerberg@linux.intel.com>
    Signed-off-by: Kirill A. Shutemov <kirill.shutemov@linux.intel.com>
    Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/pci/hotplug/acpiphp.h b/drivers/pci/hotplug/acpiphp.h
index dac836c..26100f5 100644
--- a/drivers/pci/hotplug/acpiphp.h
+++ b/drivers/pci/hotplug/acpiphp.h
@@ -144,16 +144,12 @@ struct acpiphp_attention_info
  struct module *owner;
 };
 
-/* PCI bus bridge HID */
-#define ACPI_PCI_HOST_HID  "PNP0A03"
-
 /* ACPI _STA method value (ignore bit 4; battery present) */
 #define ACPI_STA_ALL   (0x0000000f)
 
 /* slot flags */
 
-#define SLOT_ENABLED  (0x00000002)
-#define SLOT_MULTIFUNCTION (0x00000004)
+#define SLOT_ENABLED  (0x00000001)
 
 /* function flags */
 
-- 
1.7.1